### DISCUSSION TITLE
Welcome to JSON Discussions!

### DISCUSSION THREAD

**Discussion by nlohmann** [nlohmann/json#2218] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dota17** [comment]
Hello everyone！
I am a software developer. My main research area is data serialization. I usually spend most of my time in the open source community. I like the open and free communication atmosphere of the community. I am fascinated by this working mode. Before that, I was the maintainer of the json-c and jsoncpp communities, responsible for the daily maintenance of the community, and I am also an important contributor to other communities. 
Needless to say, this community is a very excellent community on GitHub, the infrastructure of the community is very perfect, and the author's daily management is very careful. At the end of last year, I was attracted by this community, so I began to participate in this community.
Up to now, I have submitted around 20 PRs, fixed some bugs, and made some improvement of this library. 

So, I want to know if I can be a member of this community, and if I can, it will be a great honor.

**Reply by nlohmann** [reply]
Thank you @dota17 for your contributions! Great to have you on board!
